File: www/src/mocks/basic_info.js
const BasicInfo = {
    code: 0,
    msg: "",
    data: {
        streamer: {
            uid: 12345,
            name: "主播昵称",
            icon: "/icon.png",
        }
    },
}

export default {
    'get|^/api/room/basic\\?room_id=[0-9]+$': opt => BasicInfo,
};

File: server/bs/basic_info.go
package bs

type BasicInfoRequest struct {
	RoomID int `form:"room_id"`
}

type BasicInfoResponse struct {
	Streamer struct {
		UID  int64  `json:"uid"`
		Name string `json:"name"`
		Icon string `json:"icon"`
	} `json:"streamer"`
}
